app/pods/publications/publication/documents/controller.js

```diff
@@ -57,4 +47,69 @@ export default class PublicationDocumentsController extends Controller {
   toggleFolderCollapse(piece) {
     piece.set('collapsed', !piece.collapsed);
   }
+
+  @action
+  uploadPiece(file) {
```

Comment:
een spinner zoals de simple-spinner zou hier mss kunnen aangeven dat het uploaden bezig is.
Als je ene grote file upload zie je "0 bestanden van de 1 opgeladen" maar niet echt een indicatie dat er iets bezig is.

Reply:
Het effectieve uploaden gebeurt in de file-uploader,
Deze code gaat de nodige objecten aanmaken om te kunnen tonen in de tabel onder de uploader

Reply:
Die progress bar tonen, moet dat in dit ticket?
![image](https://user-images.githubusercontent.com/22245223/101160813-ef3c4780-362f-11eb-9ad1-192b63d3520d.png)